### DISCUSSION TITLE
Docker image for 6.9.0?

### DISCUSSION THREAD

**Discussion by flickerfly** [RocketChat/Rocket.Chat#32573] [General]
I saw 6.9.0 came out and went over to docker to get the image, but it doesn't appear present. What should my expectations be regarding the availability of that container?

https://hub.docker.com/_/rocket.chat

**Comment by rvva** [comment]
Rocket.chat deploys images first on its own repository. If you want to use the latest image, here is the best source to download it. 
`docker pull registry.rocket.chat/rocketchat/rocket.chat:latest`.

If you want to list all tags, images available in the repo, use curl or skopeo tool:

```
curl -s https://registry.rocket.chat/v2/rocketchat/rocket.chat/tags/list | jq 
skopeo list-tags docker://registry.rocket.chat/rocketchat/rocket.chat
```
At the moment when I write this, there is a 6.11.1 image available in registry.rocket.chat and a 6.9.2 image available in dockerhub.

**Comment by reetp** [comment]
Never, EVER use latest tags unless you enjoy large and unexpected breakages.

See here for why.

https://gist.github.com/reetp/ddd0580ab9f9397715da1c14cd85a73c#file-readme-md

Both registry and docker hub images are updated but note that hub is maintained internally on an ad hoc basis and often lags behind the registry version.
